server/discussionAnchor/__tests__/queries.test.ts

```diff
@@ -26,6 +26,8 @@ setup(beforeAll, async () => {
 	await models.resolve();
 });
 
+teardown(afterAll);
```

Comment:
ive added `teardown` everywhere `setup` is used to close the connection with `sequelize`